src/Workspaces/Core/Portable/Storage/PersistenceStorageServiceFactory.cs

```diff
@@ -11,17 +11,29 @@
 
 // When building for source-build, there is no sqlite dependency
 #if !DOTNET_BUILD_FROM_SOURCE
+using Microsoft.CodeAnalysis.SQLite.v2;
 #endif
 
 namespace Microsoft.CodeAnalysis.Storage
 {
     [ExportWorkspaceServiceFactory(typeof(IPersistentStorageService), ServiceLayer.Desktop), Shared]
     internal class PersistenceStorageServiceFactory : IWorkspaceServiceFactory
     {
+#if !DOTNET_BUILD_FROM_SOURCE
+        private readonly SQLiteConnectionPoolService _connectionPoolService;
+#endif
```

Comment:
for my understanding, this is because in `DOTNET_BUILD_FROM_SOURCE` we just have no persistence service at all?

Reply:
Right, there is no SQLite reference.